File: Module1/8-8/OverloadingEx.java
package com.a88;

public class OverloadingEx 
{
	
	int cal(int a,int b)
	{
		return a+b;
	}
	double cal(double a,double b)
	{
		return a*b;
	}
	
	public static void main(String[] args) 
	{
		OverloadingEx o1 =new OverloadingEx();
		
		System.out.println(o1.cal(5,6));//addition
		System.out.println(o1.cal(2.99,3.99));//multiplication
		
		
	}
}

File: Module1/Kotlin/26-8/OverloadingEx.kt
class OverloadingEx
{

    fun add(a:Int,b:Int):Int
    {
        return a+b
    }
    fun add(a:Int,b:Int,c:Int):Int
    {
        return  a*b*c
    }

}

fun main()
{
    var o1 =OverloadingEx()
    println(o1.add(5,6))
    println(o1.add(5,6,7))

}